x/tss/keeper/grpc_query.go

```diff
@@ -114,15 +114,15 @@ func (q Querier) getSerializedMultisigKeys(ctx sdk.Context, key exported.Key, va
 		return nil, false
 	}
 
-	pubkeys, ok := q.keeper.GetMultisigPubKeysByValidator(ctx, key.ID, valAddress)
-	if !ok {
+	pubKeys, ok := q.keeper.GetMultisigPubKeysByValidator(ctx, key.ID, valAddress)
```

Comment:
A proper (but consensus breaking) fix will be to make sure this method returns false if no keys are found for this validator.

Reply:
it's weird that it returns true